packages/dendron-plugin-views/src/hooks/index.tsx

```diff
@@ -94,7 +94,10 @@ export const useMermaid = ({
     if (config && ConfigUtils.getPreview(config)?.enableMermaid) {
       mermaid.initialize({
         startOnLoad: true,
-        theme: themeType === "light" ? "forest" : "dark",
```

Comment:
What kind of problem is it causing? A runtime error?

Reply:
The type definition says I should be able to get the enum by doing `mermaid.mermaidApi.THEME.light` (or something like that, not looking it up rn). But if I do that `mermaid.mermaidApi.THEME` is `undefined`.